cmd/repo_link.go

```diff
@@ -0,0 +1,238 @@
+	"github.com/Scalingo/cli/repolink"
+	"github.com/Scalingo/go-scalingo"
+)
+
+var (
+	repoLinkShowCommand = cli.Command{
+		Name:     "repo-link",
+		Category: "Repo Link",
+		Usage:    "Show repo link linked with your app",
+		Flags:    []cli.Flag{appFlag},
+		Description: ` Show repo link linked with your application:
+	$ scalingo --app my-app repo-link
+
+		# See also 'repo-link-create', 'repo-link-update', 'repo-link-delete', 'repo-link-manual-deploy', 'repo-link-manual-review-app'`,
+		Action: func(c *cli.Context) {
+			if len(c.Args()) != 0 {
+				cli.ShowCommandHelp(c, "repo-link")
+				return
+			}
+
+			currentApp := appdetect.CurrentApp(c)
+			err := repolink.Show(currentApp)
+			if err != nil {
+				errorQuit(err)
+			}
+		},
+		BashComplete: func(c *cli.Context) {
+			_ = autocomplete.CmdFlagsAutoComplete(c, "repo-link")
+		},
+	}
+
+	repoLinkCreateCommand = cli.Command{
+		Name:     "repo-link-create",
+		Category: "Repo Link",
+		Flags: []cli.Flag{
+			appFlag,
+			cli.StringFlag{Name: "branch", Usage: "Branch used in auto deploy"},
+			cli.StringFlag{Name: "auto-deploy", Usage: "Enable auto deploy of application after each branch change"},
+			cli.StringFlag{Name: "deploy-review-apps", Usage: "Enable auto deploy of review app when new pull request is opened"},
+			cli.StringFlag{Name: "destroy-on-close", Usage: "Auto destroy review apps when pull request is closed"},
+			cli.StringFlag{Name: "hours-before-destroy-on-close", Usage: "Time delay before auto destroying a review app when pull request is closed"},
+			cli.StringFlag{Name: "destroy-on-stale", Usage: "Auto destroy review apps when no deploy/commits has happened"},
+			cli.StringFlag{Name: "hours-before-destroy-on-stale", Usage: "Time delay before auto destroying a review app when no deploy/commits has happened"},
+		},
+		Usage: "Create a repo link between your scm integration and your app",
+		Description: ` Create a repo link between your scm integration and your application:
+	$ scalingo --app my-app repo-link-create <integration-name> <repo-url> [options]
+									   OR
+	$ scalingo --app my-app repo-link-create <integration-uuid> <repo-url> [options]
+
+	List of available integrations:
+	- github => GitHub.com
+	- github-enterprise => GitHub Enterprise (private instance)
+	- gitlab => GitLab.com
+	- gitlab-self-hosted => GitLab Self-hosted (private instance)
+
+	Examples:
+	$ scalingo -a test-app repo-link-create gitlab https://gitlab.com/gitlab-org/gitlab-ce
+	$ scalingo -a test-app repo-link-create github-enterprise https://ghe.example.org/test/frontend-app --branch master --auto-deploy true
+
+		# See also 'repo-link', 'repo-link-update', 'repo-link-delete', 'repo-link-manual-deploy' and 'repo-link-manual-review-app'`,
+		Action: func(c *cli.Context) {
+			if len(c.Args()) != 2 {
+				cli.ShowCommandHelp(c, "repo-link-create")
+				return
+			}
+
+			currentApp := appdetect.CurrentApp(c)
+			integrationType := c.Args()[0]
+			integrationURL := c.Args()[1]
```

Comment:
Should be `--url`
The type becomes optional if github.com or gitlab.com in the URL